### PR TITLE
feat(130): Introduces SEQREPO_FD_CACHE_MAXSIZE env var

### PR DESCRIPTION
**Pull request by kazmiekr**
Introduces `SEQREPO_FD_CACHE_MAXSIZE` env var to override the internal `fd_cache_size` to allow for increased performance without forcing code changes to any SeqRepo clients. 

Example:
We are using SeqRepo in an internal library and noticed a 10x decrease in performance with seqrepo 0.6.6 because the internal `fd_cache_size` is being set to 0 ( no caching ). Rather than forcing clients of SeqRepo to set this value in code and release new versions, it'd be nice to control this value via an env var like `SEQREPO_LRU_CACHE_MAXSIZE`

Note:
Now that there are two env vars, the old var `SEQREPO_LRU_CACHE_MAXSIZE` feels incorrectly named as it's unclear which LRU cache it's controlling. I didn't want to introduce any changes to existing code, but maybe this should be renamed to something like `SEQREPO_DB_CACHE_MAXSIZE` or something to indicate what is being cached and deprecate `SEQREPO_LRU_CACHE_MAXSIZE` or allow the one env to control all the caches?

Also is it worth providing a better default other than 0 (no cache)? Something small'ish like ~25 or so would at least give a significant performance boost without risking resource exhaustion

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,6 +151,10 @@ SEQREPO_LRU_CACHE_MAXSIZE sets the lru_cache maxsize for the sqlite
 query response caching. It defaults to 1 million but can also be set to
 "none" to be unlimited.
 
+SEQREPO_FD_CACHE_MAXSIZE sets the lru_cache size for file handler caching during FASTA sequence retrievals. 
+It defaults to 0 to disable any caching, but can be set to a specific value or "none" to be unlimited. Using 
+a moderate value (>10) will greatly increase performance of sequence retrieval.
+
```
Comment (andreasprlic) [inline README.md:157]: What is the downside of increasing this value?

**Comment by andreasprlic** [comment]
I think the mentioned setup issue will require a change [similar to this](https://github.com/biocommons/bioutils/pull/57/files). That appears to be a problem we need to fix across all biocommons packages.

File: README.md (hunk 1 above)
Comment (kazmiekr) [reply README.md:157]: I think the only downside is in the potential for resource exhaustion on the system? But I'd defer to the discussion in https://github.com/biocommons/biocommons.seqrepo/issues/112 for additional information. Should I add a statement that states if the value is too large, there is a risk for exhaustion?

**Review by andreasprlic — APPROVED** [review]
Thanks, LGTM

**Pull request merged**